src/main/webapp/app/app.component.html

```diff
@@ -4,12 +4,13 @@
       <div class="logo">
         <img src="assets/images/logos/coyo/logo-coyo-inversed-nav-hd.png" alt="COYO">
       </div>
-      <tiny-task-form (created)="created($event)"></tiny-task-form>
+      <tiny-task-form (created)="created()"></tiny-task-form>
+      <tiny-search-tasks [reset$]="resetSearch$" (emitted)="searchByQuery($event)"></tiny-search-tasks>
```

Comment:
Interesting that you choose a subject here

Reply:
Firstly I was thinking about using it into the component template somehow, then I thought about using some rxjs operators for some reasons. 
Eventually I did not implemented any of these ideas :) and left it as it.

Sure, it coud be just a boolean input parameter as a trigger.